web/src/beta/components/Modal/index.tsx

```diff
@@ -79,11 +82,13 @@ const Modal: React.FC<Props> = ({
         )}
         <ContentWrapper>
           {tabs.length > 0 && <Content>{tabs.find(tab => tab.active === true)?.content}</Content>}
-          <Content> {children}</Content>
-          <ButtonWrapper>
-            {tabs.find(tab => tab.active === true)?.tabButton1 ?? button1}
-            {tabs.find(tab => tab.active === true)?.tabButton2 ?? button2}
-          </ButtonWrapper>
+          {children && <Content> {children}</Content>}
```

Comment:
Double check your IDE is working with our linting? I see a gap between <Content> and {children}